### DISCUSSION TITLE
Tool enabling in chat is not sticky. The toogle seems to reset itself.

### DISCUSSION THREAD

**Discussion by Taluen79** [open-webui/open-webui#4829] [General]
# Bug Report

## Installation Method
From source
[Describe the method you used to install the project, e.g., git clone, Docker, pip, etc.]
git clone

## Environment
- **Open WebUI Version:** [e.g., v0.3.11]
- **Ollama (if applicable):** [e.g., v0.2.0, v0.1.32-rc1]
Open WebUI 0.3.15

- **Operating System:** [e.g., Windows 10, macOS Big Sur, Ubuntu 20.04]
- **Browser (if applicable):** [e.g., Chrome 100.0, Firefox 98.0]
Windows, Brave browser. The server is run in a conda environment.

**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [x] I have included the browser console logs.
- [x] I have included the Docker container logs.
- [x] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:
Enabled tool via the "+" sign in a chat should stay enabled even when clicking around the GUI.

[Describe what you expected to happen.]
Either the state of the tool is "remembered" per chat or globally. But is does not seems so. It's very confusing and annoying because you think you have enabled a tool, but it might have been disabled if you have navigated around the GUI.

## Actual Behavior:
Sometimes when I enable a tool for a model to use it and then in the middle of a chat I might look into the workspace or other settings. When I go back to the chat I noticed that the enabled tool has been disabled by itself for some reason and I have to enable it again. So far I have only tested the image_gen tool from the tool community page.

**Comment by atgehrhardt** [comment]
This is the intended behavior. The tools are disabled in new chats and when leaving existing chats.

You can use the AutoTool Filter Function to allow models to use any tools enabled in their model settings automatically: https://openwebui.com/f/hub/autotool_filter

**Reply by UberMetroid** [reply]
Is there a guide for this tool? I cant figure it out.
